src/metabase/util.clj

```diff
@@ -29,7 +30,7 @@
 ;; This is the very first log message that will get printed.
 ;; It's here because this is one of the very first namespaces that gets loaded, and the first that has access to the logger
 ;; It shows up a solid 10-15 seconds before the "Starting Metabase in STANDALONE mode" message because so many other namespaces need to get loaded
-(log/info "Loading Metabase...")
+(log/info (trs "Loading {0}..." (trs "Metabase")))
```

Comment:
You don't need the second `trs` here

Reply:
Are you sure? I want "Metabase" itself to be a translated string.

Reply:
I don't think it should be a translated string in this PR?

Reply:
Fixed